packages/account-management/src/routes/index.ts

```diff
@@ -5,7 +5,7 @@ const unAuthenticatedPages: string[] = ['/register', '/validated', '/error'];
 export const indexPage: RouteMiddleware = (context) => {
   const bundle = context.routes.url('assets-bundles');
 
-  const isUnauthenticatedPage: boolean = !!context.req.url && unAuthenticatedPages.includes(context.req.url);
+  const isUnauthenticatedPage: boolean = !!context.req.url && unAuthenticatedPages.includes(context.req.url.split('?')[0]);
```

Comment:
`context.request.URL.pathname` seems to give us what we're looking for here, and it isn't optional:

```suggestion
  const isUnauthenticatedPage: boolean = unAuthenticatedPages.includes(context.request.URL.pathname);
```

Reply:
Good spot! That's much nicer